### PR TITLE
Added the loading component

### PR DESCRIPTION
**Pull request by Vidhanvyrs**

### CODE REVIEW

**Comment by Vidhanvyrs** [comment]
Please provide feedback, if i made a mistake

**Review by anicmarko9 — COMMENTED** [review]
Thanks for contribution, everything looks good except for some small inconveniences. I will wait for them to get fixed before I merge into main. Feel free to ask me anything if you are in doubt.

File: src/components/Loading/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const Loading: React.FC = () => {
+  return (
+    <div className="loading-container">
+      <div className="loading-spinner"></div>
+    </div>
+  );
+};
+
+export default Loading;
```
[the remark this answers is not in the document]
Comment (Vidhanvyrs) [reply src/components/Loading/loading.tsx:12]: On it

File: src/index.css [hunk 2]
```diff
@@ -118,4 +118,24 @@
...
+
+  .loading-container {
+    @apply flex justify-center items-center h-screen;
+  }
+  
+  .loading-spinner {
+    @apply animate-spin border-t-4 border-blue-500 border-opacity-50 rounded-full h-16 w-16;
+    border-right: 4px solid transparent;
+    border-bottom: 4px solid transparent;
+    border-left: 4px solid transparent;
```
[the remark this answers is not in the document]
Comment (Vidhanvyrs) [reply src/index.css:130]: my mistake !! Resolving it

File: src/index.css [hunk 3]
```diff
@@ -118,4 +118,24 @@
...
+  }
+  
+  @keyframes spin {
+    0% {
+      transform: rotate(0deg);
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+  }
```
[the remark this answers is not in the document]
Comment (Vidhanvyrs) [reply src/index.css:140]: ok

**Comment by Vidhanvyrs** [comment]
okay all issues are resolved now i think

**Pull request merged**